src/components/Cart/Cart.tsx

```diff
@@ -0,0 +1,47 @@
+import './Cart.scss';
+import LeftIcon from '../../icons/left.svg';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+
+export const Cart = () => {
+  const [cartItems, setCartItems] = useState([]);
+  const navigate = useNavigate();
+
+  const handleCheckout = () => {
+    const userConfirmed = confirm(
+      'Checkout is not implemented yet. Do you want to clear the Cart?',
+    );
+
+    if (userConfirmed) {
+      setCartItems([]);
+      navigate('/phones');
+    }
+  };
+
+  return (
+    <>
+      <div className="cart">
+        <NavLink to="#">
+          <img className="cart__icon" src={LeftIcon} alt="left" />
+        </NavLink>
+        <a className="cart__back">Back</a>
```

Comment:
Wrap not only a button but also a text since I personally and I think most people will click on a "Back"

Reply:
"Back" wrapped